styles.css

```diff
@@ -46,6 +46,12 @@ body {
   user-select: none;
 }
 
+.nav-bar__toggle-completed--logo {
```

Comment:
It's working perfectly, thanks Mark!

The only thing is that if you filter out completed tasks and then look at the page the emoji doesn't really make it easy to understand whether they are filtered out or not.

I tried to have a look around for logos that would help and found a pair that should work well, also found a sun and moon in the same style!

Can you try these? :

&#9744; - for the filter when it's off
&#9745; - for the filter when it's on

&#9788; - to replace the current sun logo
&#9789; - to replace the current moon logo

this is the origin of those: https://www.w3schools.com/charsets/ref_utf_symbols.asp